Easydict/objc/MMKit/Log/MMLog.m

```diff
@@ -29,39 +29,27 @@ + (void)load {
     dispatch_once(&onceToken, ^{
         // 系统日志、控制台 格式设置
         MMConsoleLogFormatter *consoleFormatter = [MMConsoleLogFormatter new];
-        if (@available(macOS 10.12, *)) {
-            [DDOSLogger sharedInstance].logFormatter = consoleFormatter;
-        } else {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-            [DDASLLogger sharedInstance].logFormatter = consoleFormatter;
-#pragma clang diagnostic pop
-        }
+        [DDOSLogger sharedInstance].logFormatter = consoleFormatter;
     });
 }
 
 + (void)configDDLog:(DDLog *)ddlog name:(NSString *)name {
     // https://github.com/CocoaLumberjack/CocoaLumberjack/blob/master/Documentation/GettingStarted.md
     NSCAssert(name.length, @"MMLog: 日志 name 不能为空");
 
-    // 系统日志、控制台
-    if (@available(iOS 10.0, *)) {
-        [ddlog addLogger:[DDOSLogger sharedInstance]];
-    } else {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wdeprecated-declarations"
-        [ddlog addLogger:[DDASLLogger sharedInstance]];
-#pragma clang diagnostic pop
-    }
+    // terminal, use os_log
+    [ddlog addLogger:[DDOSLogger sharedInstance]];
 
     // 文件输出
     MMFileLogFormatter *fileFormatter = [MMFileLogFormatter new];
     DDLogFileManagerDefault *fileManager = [[DDLogFileManagerDefault alloc] initWithLogsDirectory:[self logDirectoryWithName:name]];
-    fileManager.maximumNumberOfLogFiles = 200;
+    fileManager.maximumNumberOfLogFiles = 10;
     fileManager.logFilesDiskQuota = 20 * 1024 * 1024;
```

Comment:
Sorry, I didn't notice before that there is already a file size limit here, in that case we don't need to limit the number of files.

Reply:
ok, changed it to 100.